src/components/Box/__stories__/Box.stories.module.scss

```diff
@@ -0,0 +1,62 @@
+@import "../../../styles/themes";
+
+.mondayStorybookBoxWrapper {
```

Comment:
it's not critical, but for next time: the monday prefix is not required on css modules class names :)

Reply:
done